### PR TITLE
Fixed issues compalined by remark-lint

### PR DESCRIPTION
**Pull request by m-linner-ericsson**
### Description of the Change
In our other project we are using [remark-lint](https://github.com/remarkjs/remark-lint) to verify the correctness of our markdowns. This pull request fixes some issues found, making the markdown more correct.

### Benefits
Markdown is more correct

### Sign-off

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by: Mattias Linnér mattias.linner@ericsson.com

### CODE REVIEW

**Comment by m-linner-ericsson** [comment]
Waiting for somebody with merge permission to handle this pr

**Review by henning-roos — CHANGES_REQUESTED** [review]
Please remove CODE_OF_CONDUCT.md and CONTRIBUTING.md from this PR since these files have moved to https://github.com/eiffel-community/.github. See https://github.com/eiffel-community/eiffel-repository-template/pull/4 for more information regarding this. 

Please, feel free to make a PR with the same changes to CODE_OF_CONDUCT.md and CONTRIBUTING.md on https://github.com/eiffel-community/.github instead.

The changes in the README.md looks good!

**Comment by m-linner-ericsson** [comment]
@henning-roos  I fixed your comments in e660256

**Pull request merged**
